docs/_static/vdr/v1.yaml

```diff
@@ -150,6 +150,89 @@ paths:
             text/plain:
               schema:
                 type: string
+  /internal/vdr/v1/did/{did}/verificationmethod:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example:
+          - "did:nuts:1234"
+        schema:
+          type: string
+    post:
+      summary: Creates a new verificationMethod and adds it to the DID document
+      operationId: addNewVerificationMethod
+      tags:
+        - DID
+      responses:
+        "201":
+          description: "New verification method has been created and added successfully. Returns the DID document."
+          content:
+            application/json+did-document:
+              schema:
+                $ref: '#/components/schemas/VerificationMethod'
+        "403":
+          description: Verification method could not be added because the DID is not managed by this node.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: Corresponding DID document could not be found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+  /internal/vdr/v1/did/{did}/verificationmethod/{kid}:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example:
+          - "did:nuts:1234"
+        schema:
+          type: string
+      - name: kid
+        in: path
+        description: URL encoded DID identifying the verification method.
+        required: true
+        example:
+          - "did:nuts:1234#abc"
+        schema:
+          type: string
+    delete:
+      summary: Delete a specific verification method
+      description: Removes the verification method from the DID Document. Revokes the public key in the keystore.
```

Comment:
*revoke* or just remove from DID document? What happens to the private key?

Reply:
Every public key is stored under its key-id. It is theoretically possible to have multiple verification methods with different key ids for the same private key because of different DID prefixes. This specific public key will be revoked. The private key does not have any expiraton properties. The private key will not be altered.